newton/_src/viewer/viewer_viser.py

```
@@ -0,0 +1,765 @@
+
+        if is_sphinx_build():
+            # Sphinx build - use static HTML with viser player
+            # The recording path needs to be relative to the viser index.html location
+            # which is at _static/viser/index.html
+
+            # Find the _static portion of the path
+            static_idx = recording_str.find("_static/")
+            if static_idx == -1:
+                raise ValueError(
+                    f"Recordings that are supposed to appear in the Sphinx documentation must be stored in docs/_static/, but the path {recording_str} does not contain _static/"
+                )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)